pkg/operators/kubenameresolver/kubenameresolver.go

```diff
@@ -134,6 +138,150 @@ func (m *KubeNameResolverInstance) EnrichEvent(ev any) error {
 	return nil
 }
 
+func (k *KubeNameResolver) GlobalParams() api.Params {
+	return nil
+}
+
+func (k *KubeNameResolver) InstanceParams() api.Params {
+	return nil
+}
+
+func (k *KubeNameResolver) Priority() int {
+	return priority
+}
+
+type k8sAccesors struct {
+	PodName   datasource.FieldAccessor
+	Namespace datasource.FieldAccessor
+	PodIP     datasource.FieldAccessor
+	HostIP    datasource.FieldAccessor
+	PodLabels datasource.FieldAccessor
+}
+
+func (k *KubeNameResolver) InstantiateDataOperator(gadgetCtx operators.GadgetContext, instanceParamValues api.ParamValues) (operators.DataOperatorInstance, error) {
+	logger := gadgetCtx.Logger()
+	accessors := make(map[datasource.DataSource]k8sAccesors)
+	for _, ds := range gadgetCtx.GetDataSources() {
+		logger.Debugf("KubeNameResolverOperator inspecting datasource %q", ds.Name())
+
+		if enable, ok := ds.Annotations()["kubenameresolver.enable"]; !ok || enable != "true" {
```

Comment:
super minor nit: you can compare directly with the string:

```suggestion
		if ds.Annotations()["kubenameresolver.enable"] != "true" {
```